packages/dev/core/src/FlowGraph/flowGraphContext.ts

```diff
@@ -1,14 +1,39 @@
+import { serialize } from "../Misc/decorators";
+import { RandomGUID } from "../Misc/guid";
 import type { Scene } from "../scene";
 import type { FlowGraphAsyncExecutionBlock } from "./flowGraphAsyncExecutionBlock";
 import type { FlowGraphBlock } from "./flowGraphBlock";
 import type { FlowGraphDataConnection } from "./flowGraphDataConnection";
 import type { FlowGraphEventCoordinator } from "./flowGraphEventCoordinator";
+import type { FlowGraph } from "./flowGraph";
+
+function defaultValueSerializationFunction(key: string, value: any, serializationObject: any) {
+    if (value?.getClassName && value?.getClassName() === "Mesh") {
+        serializationObject[key] = {
+            name: value.name,
+            className: value.getClassName(),
+        };
+    } else {
+        serializationObject[key] = value;
+    }
+}
+
+function defaultValueParseFunction(key: string, serializationObject: any, scene: Scene) {
+    const value = serializationObject[key];
+    let finalValue;
+    if (value?.className === "Mesh") {

```

Comment:
is it only for mesh? or also for InstanceMesh, GroundMesh, AbstractMesh?

Reply:
Makes me wonder if we should have helper functions for things like that. (Not for this PR, just a thought).
isMesh, isLight, isTransfornNode ? maybe a part of tools. I'll see what's possible